packages/block-library/src/image/edit.native.js

```diff
@@ -564,8 +561,23 @@ export class ImageEdit extends Component {
 				<PanelBody title={ __( 'Link Settings' ) }>
 					{ this.getLinkSettings( true ) }
 				</PanelBody>
-				{ canImageBeFeatured &&
-					this.getFeaturedButtonPanel( isFeaturedImage ) }
+				<PanelBody
+					title={ __( 'Featured Image' ) }
+					titleStyle={ styles.featuredImagePanelTitle }
+				>
+					{ canImageBeFeatured &&
+						this.getFeaturedButtonPanel( isFeaturedImage ) }
+				</PanelBody>
+				<PanelBody>
+					<FooterMessageControl
+						label={ __(
+							'Changes to featured image will not be affected by the undo/redo buttons.'
+						) }
+						cellContainerStyle={
+							styles.setFeaturedButtonCellContainer
+						}
+					/>
+				</PanelBody>
```

Comment:
I was wondering if we really need to have the footer message under a new panel body, if the purpose is to prevent rendering a separator, you could disable it by passing the prop `separatorType="none"` to the `BottomSheet.Cell` components defined in the `getFeaturedButtonPanel` function. This way you could render the `FooterMessageControl` component in the same panel body as the featured button.

Reply:
You're right that there's no meaningful reason for the footer message to exist in its own panel body. I've gone ahead to move it within the featured panel in https://github.com/WordPress/gutenberg/pull/33057/commits/56625086337fe41cd4ad18b4240f56f37c2cf3d1, and have removed the separator as per your suggestion. :)